backend/app/utils.py

```
@@ -160,6 +160,16 @@ def verify_token(token: str = Security(oauth2_scheme)):
             headers={"WWW-Authenticate": "Bearer"},
         )
 
+def verify_reset_token(token: str) -> str:
```

Ruff (0.8.2)

Redefinition of unused `verify_reset_token` from line 108

(F811)